spec.html

```diff
@@ -15084,6 +15097,25 @@
             1. Assert: _status_ is *true*.
             1. Increment _n_ by 1.
         </emu-alg>
+        <emu-grammar>BindingRestElement : `...` BindingPattern</emu-grammar>
```

Comment:
I've copied the steps from 13.3.3.6 (IteratorBindingInitialization for |BindingRestElement : ... BindingIdentifier|). (I know that 12.14.5 and 13.3.3 use different algorithm steps for the same kind of work, but I didn't want to tackle that issue in this PR.)